client/app/admin/create-project/create-project.controller.js

```diff
@@ -30,15 +31,15 @@
          */
         vm.setWizardStep = function(wizardStep){
             if (wizardStep === 'tasks'){
-                var numberOfFeatures = drawService.getFeatures().length;
-                if (numberOfFeatures > 0){
-                    vm.AOIRequired = false;
+
+                var aoiValidationObj = projectService.validateAOI(drawService.getFeatures());
```

Comment:
See comment below as well. Renaming aoiValidationObj to aoiValidationResult as a suggestion?